doc/quick-start-guide-and-build.md

```diff
@@ -145,6 +147,29 @@ Before building _LPVS_ from source code, ensure that you have the following prer
   sudo apt install mysql-server
   ```
 
+  
+> [!NOTE]  
+> For installing SCANOSS on Ubuntu 23.04, Fedora 38, Debian 11, etc. or later versions of mentioned OSes, a few additional steps are required, as previously mentioned command : `pip3 install --require-hashes -r requirements.txt` will not work correctly. [More details](https://github.com/scanoss/scanoss.py/blob/main/PACKAGE.md#externally-managed-environments-on-linux)
+
+Installation steps:
+- Install `pipx`:
+  ```bash
+  sudo apt install pipx
+  ```
+- Install `scanoss` by using `pipx`:
+  ```bash
+  pipx install scanoss
```

Comment:
Why don't you use `--require-hashes -r requirements.txt` here?

Reply:
Please find the solution first. This is a known issue. Please refer to https://github.com/pypa/pipx/issues/359

Reply:
Seems pipx still does not have official solution for installing from a requirement file.

As confirmation of it-
recently opened PR and issue for: Preinstall from requirements file 
https://github.com/pypa/pipx/pull/1536
https://github.com/pypa/pipx/issues/1525

If this PR will be merged seems will be standard possibility to use requirement file as syntax is:
 pipx install -v --preinstall-from-file /tmp/httpx-req.txt --preinstall-from-file /tmp/attrs-req.txt dunk (it taken from  the PR)


Another way by using additional packet:
pipx install cookiecutter
pipx runpip cookicutter install -r requirements.txt
But I can not run it as some downloaded packets have other hashes than in the file of requirements.

Reply:
Got it. Will be added to the guide after pipx feature official release.